buildspec-build-invoke.yml

```diff
@@ -0,0 +1,17 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      docker: 19
+
+    commands:
+      # start docker daemon
+      - nohup /usr/local/bin/dockerd --host=unix:///var/run/docker.sock --host=tcp://127.0.0.1:2375 --storage-driver=overlay2&
```

Comment:
Curious do we have always have to start the docker daemon? or are there codebuild images which will docker started and ready to go?

Reply:
No we don't! Tested and it works!